core/src/main/scala/org/apache/spark/rpc/netty/NettyStreamManager.scala

```diff
@@ -31,6 +31,7 @@ private[netty] class NettyStreamManager(rpcEnv: NettyRpcEnv)
 
   private val files = new ConcurrentHashMap[String, File]()
   private val jars = new ConcurrentHashMap[String, File]()
+  private val dirs = new ConcurrentHashMap[String, File]()
```

Comment:
while you are at it, can you update the classdoc for this to explain the different things this manager can serve, and explain the difference between jars and files?